### ISSUE TITLE
Update `rustls` dep?

### ISSUE THREAD

**Issue by Geobert** [inejge/ldap3#76]
Hi there,

is there any plan for a release that only upgrade some deps, especially `rustls`?

I have an issue when passing a `ClientConfig` to `LdapConnSettings`:
```
note: expected struct `std::sync::Arc<rustls::client::ClientConfig>`
         found struct `std::sync::Arc<rustls::ClientConfig>`
```

**Comment by inejge** [comment]
Incompatible versions strike again, eh? :slightly_smiling_face: Yes, I'll look into releasing an updated version in the very near future.

**Comment by inejge** [comment]
Updated and published 0.10.0.
